screen/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Screen component for screencrash",
   "main": "main.js",
   "scripts": {
-    "start": "electron .",
-    "dev": "nodemon --exec electron .",
+    "start": "npx electron .",
```

Comment:
Is this needed on Windows?

Reply:
Yep, electron is not in the PATH if just running as it is (unless you have electron installed globally, because that folder is in the path. but that shouldnt be a requirement)

Reply:
Weird that npm doesn't take care of that, since it does on Linux. But if it works it works.